modules/project_cleanup/function_source/main.go

```diff
@@ -290,6 +349,35 @@ func getTagValuesServiceOrTerminateExecution(ctx context.Context, client *http.C
 	return cloudResourceManagerService.TagValues
 }
 
+///r
+//https://pkg.go.dev/cloud.google.com/go/asset/apiv1#Client.ListFeeds
+func getListFeedsRequestOrTerminateExecution(ctx context.Context, client *http.Client, organizationId string) (*assetpb.ListFeedsRequest, error) {
+	logger.Println("Creating ListFeedsRequest")
+
+	_, err := asset.NewClient(ctx, option.WithHTTPClient(client))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create Asset client: %v", err)
+	}
+
+	randomIDPattern := `^organizations/` + organizationId + `/feeds/fd-cai-monitoring-[0-9a-f]{2}$`
```

Comment:
this value, `fd-cai-monitoring-[0-9a-f]{2}` should not be in the code. this should be the values provided in the variable `target_included_feeds`